saleor/graphql/schema.graphql

```diff
@@ -965,7 +980,7 @@ type Mutations {
   customerPasswordReset(input: CustomerPasswordResetInput!): CustomerPasswordReset
   customerRegister(input: CustomerRegisterInput!): CustomerRegister
   customerUpdate(id: ID!, input: CustomerInput!): CustomerUpdate
-  customerAddressCreate(input: AddressInput!): CustomerAddressCreate
+  customerAddressCreate(input: DefaultAddressInput!): CustomerAddressCreate
```

Comment:
Instead of introducing a new type which is almost the same as `AddressInput`, we could change the signature of the mutation to: `customerAddressCreate(input: AddressInput, type: AddressTypeEnum): CustomerAddressCreate `. Additionally, we should provide a description for the `type` argument e.g. "A type of address. If provided, the new address will be automatically assigned as the customer's default address of that type."

Reply:
Sounds better, thanks, will do!

Reply:
I remembered why I design like this now, it looks like ```ModelMutation``` only supports ID and input argument, adding more arguments requires override some methods...